lib/class-wp-sidebar-block-editor-control.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class WP_Sidebar_Block_Editor_Control extends WP_Customize_Control {
+	public $type = 'sidebar_block_editor';
+
+	public function enqueue() {
+		gutenberg_widgets_init( 'gutenberg_customizer' );
+	}
+
+	public function render_content() {
+		?>
+		<input
```

Comment:
Where is this `<input>` used? I don't think it's necessary. Ideally the `WP_Customize_Control` should just render a container that the React app mounts itself in.